tools/tab_checker

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+pr_id=$1
+files=$(curl https://api.github.com/repos/os-autoinst/scripts/pulls/${pr_id}/files | jq -c '.[].filename' | sed 's/\"//g')
+checker() {
+for n in "$files"; do
```

Comment:
This will not work correctly. The quotes will prevent argument splitting. It would be better to use arrays.